courses/models.py

```diff
@@ -91,6 +91,12 @@ def live(self):
         """Applies a filter for Course runs with live=True"""
         return self.filter(live=True)
 
+    def is_available(self):
```

Comment:
Please rename to `available`. That would keep it in line with `live` and `with_text_id`.